docs/proposals/kubelet-eviction.md

```diff
@@ -69,6 +69,7 @@ The `kubelet` will support the ability to trigger eviction decisions on the foll
 | Eviction Signal  | Description                                                                     |
 |------------------|---------------------------------------------------------------------------------|
 | memory.available | memory.available := node.status.capacity[memory] - node.stats.memory.workingSet |
+| disk.available   | disk.available := node.stats.fs.available |
```

Comment:
Should image and root file system thresholds be expressed separately in the case where they are not unified?